src/bootstrap/compile.rs

```diff
@@ -1161,9 +1161,16 @@ impl Step for Assemble {
             let dst_exe = exe("rust-llvm-dwp", target_compiler.host);
             let llvm_config_bin = builder.ensure(native::Llvm { target: target_compiler.host });
             if !builder.config.dry_run {
-                let llvm_bin_dir = output(Command::new(llvm_config_bin).arg("--bindir"));
-                let llvm_bin_dir = Path::new(llvm_bin_dir.trim());
-                builder.copy(&llvm_bin_dir.join(&src_exe), &libdir_bin.join(&dst_exe));
+                if target_compiler.host == build_compiler.host {
+                    let llvm_bin_dir = output(Command::new(llvm_config_bin).arg("--bindir"));
+                    let llvm_bin_dir = Path::new(llvm_bin_dir.trim());
+                    builder.copy(&llvm_bin_dir.join(&src_exe), &libdir_bin.join(&dst_exe));
+                } else {
+                    // if cross-compiling don't copy the llvm directory for the target instead
+                    let llvm_bin_dir = format!("build/{}/llvm/build/bin", target_compiler.host);
```

Comment:
Hmm, would this work for the host too? It would be nice to do this unconditionally instead of making it different, people use download-ci-llvm a lot so this means it would get a lot more testing.

Reply:
I think it's better not to, as technically if not cross-compiling it's more correct to use `llvm-config` to have llvm get the path to itself (which, while better, isn't possible when the host and the target aren't the same)